tcms/testruns/models.py

```diff
@@ -256,9 +257,21 @@ class TestExecution(TCMSActionModel):
         on_delete=models.CASCADE,
     )
     case_text_version = models.IntegerField()
-    close_date = models.DateTimeField(null=True, blank=True)
+    start_date = models.DateTimeField(null=True, blank=True, db_index=True)
+    stop_date = models.DateTimeField(null=True, blank=True, db_index=True)
```

Comment:
These fields also require TestExecutionFactory to be updated otherwise they cause the tests to fail.